quinn-udp/src/windows.rs

```diff
@@ -107,23 +107,35 @@ impl UdpSocketState {
             )?;
         }
 
-        // Opportunistically try to enable GRO
-        _ = set_socket_option(
-            &*socket.0,
-            WinSock::IPPROTO_UDP,
-            WinSock::UDP_RECV_MAX_COALESCED_SIZE,
-            // u32 per
-            // https://learn.microsoft.com/en-us/windows/win32/winsock/ipproto-udp-socket-options.
-            // Choice of 2^16 - 1 inspired by msquic.
-            u16::MAX as u32,
-        );
-
         let now = Instant::now();
         Ok(Self {
             last_send_error: Mutex::new(now.checked_sub(2 * IO_ERROR_LOG_INTERVAL).unwrap_or(now)),
         })
     }
 
+    /// Enable or disable receive offloading.
```

Comment:
Are there constraints on when this could/should be called?

Reply:
As in which state of the lifecycle of the socket? As far as I am aware, there are no constraints, e.g. you could toggle it between each read. That said, there might be some delay, given that, I assume, the NIC might have already coalesced some datagrams.

Reply:
Okay, that's fine then.